content/blog/2023/volunteering.md

```diff
@@ -0,0 +1,160 @@
+Title: Help us make PyCon India 2023 even more amazing
```

Comment:
I'm not sure about this.
We've had almost the same title for 2020, and 2021 too 😆 

But I'm fine with it if you are.

Reply:
oh is it :D I'll change it.